### PR TITLE
9.0 [MIG] partner_contact_job_position migration

### PR DESCRIPTION
**Pull request by cubells**

### CODE REVIEW

File: partner_contact_job_position/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* `Antiun Ingeniería S.L. <http://www.antiun.com>`_: Icon.
```
Comment (pedrobaeza) [inline partner_contact_job_position/README.rst]: This icon link is not correct.

File: partner_contact_job_position/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# © 2016 Tecnativa S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner job position",
+    "summary": "Categorize job positions for contacts",
+    "version": "9.0.1.0.0",
+    'category': 'Customer Relationship Management',
+    "website": "http://www.antiun.com",
+    'author': 'Serv. Tecnolog. Avanzados - Pedro M. Baeza, '
```
Comment (pedrobaeza) [inline partner_contact_job_position/__openerp__.py]: Switch to `Tecnativa` both website and author.

File: partner_contact_job_position/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+class ResPartnerJobPosition(models.Model):
+    _name = "res.partner.job_position"
+    _order = "parent_left"
+    _parent_order = "name"
+    _parent_store = True
+    _description = "Job position"
+
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(
```
Comment (pedrobaeza) [inline partner_contact_job_position/models/res_partner.py:27]: Check recursion

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Choose another icon different, because this one is used in partner_sector.

Squash "OCA transbot" and the "en.po" commits  together.

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: partner_contact_job_position/models/res_partner.py (hunk 3 above)
Comment (pedrobaeza) [reply partner_contact_job_position/models/res_partner.py:27]: And add a test for that

Comment (pedrobaeza) [reply partner_contact_job_position/models/res_partner.py:27]: Well, if you use something like https://github.com/odoo/odoo/blob/af865c2e9ea1dc1890ef33add9a6e4bc657068e0/addons/hr/hr.py#L264, there's no need for the test, as this method is already tested.

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Review by yajo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cubells, please make the changes I requested.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**